src/utils/verse.ts

```diff
@@ -44,6 +46,17 @@ export const getVerseAndChapterNumbersFromKey = (verseKey: string): [string, str
   return [splits[0], splits[1]];
 };
 
+/**
+ * Split the word's location and get the surahNumber, verseNumber and wordNumber.
+ *
+ * @param {String} wordLocation the word location {surahNumber}:{verseNumber}:{wordNumber}
+ * @returns {[String, String, String]}
+ */
+export const getWordDataByLocation = (wordLocation: string): [string, string, string] => {
```

Comment:
getWordLocationByString?

Reply:
Or we can get word location from BE as well.